src/config/print/config.md

```diff
@@ -32,3 +32,13 @@ you can install the `cups-pk-helper` package to allow unprivileged users to use
 While `system-config-printer` is shown here, your desktop environment may have a
 native printer dialog which may be found by consulting the documentation for
 your DE.
+
+# Troubleshooting
+
+## USB printer not shown
```

Comment:
Use `##` and `###` for the headers, because this is now inside the Configuring section.